sonar-orchestrator/src/main/java/com/sonar/orchestrator/container/SonarDistribution.java

```diff
@@ -41,6 +41,8 @@ public final class SonarDistribution {
   private boolean activateLicense;
   private boolean emptySonarProperties = false;
   private boolean keepBundledPlugins = false;
+
+  private List<String> bundledPluginsToKeep = new ArrayList<>();
```

Comment:
A Set should be a better data structure in this case I think, it would ensure you have a unique list of plugins

Reply:
fixed